### PR TITLE
Enchilada manual display card - variables

### PR DESCRIPTION
**Pull request by Calanthe**
Continuation of the https://github.com/guardian/frontend/pull/11984. In DFP you can define now color, logo position, header + text position and header + text font size:

screen shot 2016-02-17 at 14 50 36

By changing those variables, author will have more impact on how the ad looks like:
Nike example:
screen shot 2016-02-17 at 14 30 55

HSBC example:
screen shot 2016-02-17 at 14 34 11

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/commercial/creatives/gu-style-comcontent.js [hunk 1]
```diff
@@ -25,12 +25,25 @@ define([
     var GustyleComcontent = function ($adSlot, params) {
         this.$adSlot = $adSlot;
         this.params  = params;
+        this.contentPosition = {
+            bottom: 'bottom: 30px;',
+            top: 'top: 30px;'
+        };
+        this.fontSize = {
```
Comment (uplne) [inline static/src/javascripts/projects/common/modules/commercial/creatives/gu-style-comcontent.js]: Why did you chose to go the inline styles way?

I'd prefer a set of classes:

Color:
default is black and for example `.gu-display-color--light` can be white color

Logo position:
`.gu-display-position--bottom-right`

You can see the pattern. To me it seems cleaner and more less coupled CSS with JS.

Comment (Calanthe) [inline static/src/javascripts/projects/common/modules/commercial/creatives/gu-style-comcontent.js]: Yeah you are right. I just followed the pattern used by fluid ads but now I realised that it's not needed here.

File: static/src/javascripts/projects/common/modules/commercial/creatives/gu-style-comcontent.js [hunk 2]
```diff
@@ -25,12 +25,25 @@ define([
...
 
     GustyleComcontent.prototype.create = function () {
         var externalLinkIcon = svgs('externalLink', ['gu-external-icon']),
             templateOptions = {
-                clickMacro: this.params.clickMacro,
+                articleContentColor: this.params.articleContentColor === 'white' ? 'gu-display__content-color--bright ' : 'gu-display__content-color--dark',
+                articleContentPosition: this.contentPosition[this.params.articleContentPosition],
+                articleHeaderFontSize: this.fontSize[this.params.articleHeaderFontSize],
+                articleTextFontSize: this.fontSize[this.params.articleTextFontSize],
+                brandLogoPosition: this.params.brandLogoPosition === 'bottom-right' ? 'gu-display__logo-pos--bottom-right' : 'gu-display__logo-pos--top-left',
```
Comment (uplne) [inline static/src/javascripts/projects/common/modules/commercial/creatives/gu-style-comcontent.js]: Do you even need to do this check? 'gu-display__logo-pos--' + this.params.brandLogoPosition

File: static/src/javascripts/projects/common/modules/commercial/creatives/gu-style-comcontent.js [hunk 3]
```diff
@@ -25,12 +25,25 @@ define([
...
+            regular: 'gu-display__content-size--regular',
+            big: 'gu-display__content-size--big'
+        };
     };
 
     GustyleComcontent.prototype.create = function () {
         var externalLinkIcon = svgs('externalLink', ['gu-external-icon']),
             templateOptions = {
-                clickMacro: this.params.clickMacro,
+                articleContentColor: this.params.articleContentColor === 'white' ? 'gu-display__content-color--bright ' : 'gu-display__content-color--dark',
```
Comment (uplne) [inline static/src/javascripts/projects/common/modules/commercial/creatives/gu-style-comcontent.js]: If you change this in DFP to bright and dark then you can do the same as I've mentioned below

**Comment by uplne** [comment]
:+1:

**Pull request merged**